Samples/SvgConsole/Program.cs

```diff
@@ -0,0 +1,205 @@
+            {
+                var inputPath = paths[i];
+                var outputFile = settings.OutputFiles != null ? settings.OutputFiles[i] : null;
+                try
+                {
+                    string outputPath = string.Empty;
+
+                    if (outputFile != null)
+                    {
+                        outputPath = outputFile.FullName;
+                    }
+                    else
+                    {
+                        var inputExtension = inputPath.Extension;
+                        outputPath = inputPath.FullName.Remove(inputPath.FullName.Length - inputExtension.Length) + ".png";
+                        if (settings.OutputDirectory != null && !string.IsNullOrEmpty(settings.OutputDirectory.FullName))
+                        {
+                            outputPath = Path.Combine(settings.OutputDirectory.FullName, Path.GetFileName(outputPath));
+                        }
+                    }
+
+                    Directory.SetCurrentDirectory(Path.GetDirectoryName(inputPath.FullName));
+
+                    Save(inputPath, outputPath, settings.Width, settings.Height);
+                }
+                catch (Exception ex)
+                {
+                    Log($"Error: {inputPath.FullName}");
+                    Error(ex);
+                }
+            }
+        }
+
+        static async Task<int> Main(string[] args)
+        {
+            var optionInputFiles = new Option(new[] { "--inputFiles", "-f" }, "The relative or absolute path to the input files")
+            {
+                Argument = new Argument<FileInfo[]>(getDefaultValue: () => null)
+            };
+
+            var optionInputDirectory = new Option(new[] { "--inputDirectory", "-d" }, "The relative or absolute path to the input directory")
+            {
+                Argument = new Argument<DirectoryInfo>(getDefaultValue: () => null)
+            };
+
+            var optionOutputDirectory = new Option(new[] { "--outputDirectory", "-o" }, "The relative or absolute path to the output directory")
+            {
+                Argument = new Argument<DirectoryInfo>(getDefaultValue: () => null)
+            };
+
+            var optionOutputFiles = new Option(new[] { "--outputFiles" }, "The relative or absolute path to the output files")
+            {
+                Argument = new Argument<FileInfo[]>(getDefaultValue: () => null)
+            };
+
+            var optionWidth = new Option(new[] { "--width" }, "The output image width override")
+            {
+                Argument = new Argument<float?>(getDefaultValue: () => null)
+            };
+
+            var optionHeight = new Option(new[] { "--height" }, "The output image height override")
+            {
+                Argument = new Argument<float?>(getDefaultValue: () => null)
+            };
+
+            var rootCommand = new RootCommand()
+            {
+                Description = "Converts a svg file to an encoded png image."

```

Comment:
`Converts svg files to encoded png images.`  :thumbsup:

Reply:
Done.